open-company/open-company-web
# NUX 2.0

Cards:
- NUX: https://trello.com/c/8XmeTVtv
- Login restyle: https://trello.com/c/MieYdhH5

To test:
- login with a normal user, do you NOT see the NUX? good
- invite a user, do you NOT see the NUX when you signup? good
- signup as a brand new user (no email domain or already used slack org), do you see the NUX? good
- logout and log back in, do you NOT see the NUX? good
- check that the NUX is still showing as expected on different screens (large screen and laptop screen mostly)
